src/components/Avatar.tsx

```diff
@@ -62,6 +62,7 @@ const styles = StyleSheet.create({
     marginBottom: 10,
   },
   contactName: {
+    ...typeScale.bodyXSmall,
```

Comment:
why `bodyXSmall` instead of `bodySmall`?

Reply:
ah good catch! i think that should be bodySmall indeed, TY for paying attention :D :D